app/src/pages/HomePage/HomePage.tsx

```diff
@@ -9,13 +9,23 @@ import { TokenGrid } from '../../components/TokenGrid';
 import { useGlobals } from '../../globalsContext';
 import { Token, TokenMetadata } from '../../model';
 
+enum ChainId {
+  Mainnet = 1,
+  Ropsten = 3,
+  Rinkeby = 4,
+  Goerli = 5,
+  Kovan = 42,
+}
 
 export const HomePage = (): React.ReactElement => {
-  const { requester, contract } = useGlobals();
+  const { web3, requester, contract } = useGlobals();
   const navigator = useNavigator();
   const [showBrowserError, setShowBrowserError] = React.useState<boolean>(false);
```

Comment:
since we have multiple errors now it might be nicer to have something like:
```
const [browserError, setBrowserError] = React.useState<text | null>(null);
```
(contunied below)